docs/en_US/Release.md

```diff
@@ -299,7 +299,7 @@
 * Support [Metis tuner](Tuner/MetisTuner.md) as a new NNI tuner. Metis algorithm has been proofed to be well performed for **online** hyper-parameter tuning.
 * Support [ENAS customized tuner](https://github.com/countif/enas_nni), a tuner contributed by github community user, is an algorithm for neural network search, it could learn neural network architecture via reinforcement learning and serve a better performance than NAS.
 * Support [Curve fitting assessor](Assessor/CurvefittingAssessor.md) for early stop policy using learning curve extrapolation.
-* Advanced Support of [Weight Sharing](AdvancedFeature/AdvancedNas.md): Enable weight sharing for NAS tuners, currently through NFS.
+* Advanced Support of *Weight Sharing*: Enable weight sharing for NAS tuners, currently through NFS.
```

Comment:
could you add the doc link in v0.5.0?

Reply:
* first, currently we only maintain documentation on `master` branch
* second, even v*.* branches are enabled on readthedocs, such cross branch links can only be implemented by `https://` links, which is not expected in our practice.